src/squlearn/kernel/ml/qkrr.py

```diff
@@ -160,15 +158,12 @@ def set_params(self, **params):
                 )
 
         param_dict = {}
+        valid_params = self.get_params().keys()
```

Comment:
this is the same dict for the same check again

Reply:
Yes that was a bug, that is fixed in the last commit